src/test/views/test_read_apriori.py

```diff
@@ -121,6 +179,9 @@ def test_returns_expected_pressure():
 def test_get_apriori_co2_vmr_does_not_vary(
     species, doy, latitude, source, expect
 ):
+    # CO is constant for a given altitude in the used apriori data.
+    # Check that returned data is constant for CO, but not constant
+    # for all species.
```

Comment:
```suggestion
    # CO2 is constant for a given altitude in the used apriori data.
    # Check that returned data is constant for CO2, but not constant
    # for all species.
```
eller? Det ser ut som att CO varierar men CO2 är konstant.

Reply:
tack